light-client/starknet-storage-verifier/src/storage/validate.rs

```diff
@@ -0,0 +1,156 @@
+use indexmap::IndexMap;
+use starknet::core::crypto::pedersen_hash;
+use starknet::core::types::{ContractsProof, Felt, MerkleNode, StorageProof};
```

Comment:
maybe we don't need the full `starknet` dependencies. We can just use `starknet_crypto` and `starknet_types_core`.

Reply:
Done in d75ee5bb7cf176a4f9ebe3e2552efc43f1f3eb5e